backend/api/av.Local.json

```diff
@@ -0,0 +1,5 @@
+{
+  "Av": {
+    "DisableScan": "true"
```

Comment:
In general most devs will not ever have a use for the av scan. For local dev the only reason you would want this is if you are working on the av code itself, which won't happen often. As a result the default for local is off.

Reply:
nit: would prefer if config option is named in the positive - ie "EnableScan" vs "DisableScan"
not a big deal though